examples/all-clusters-app/ameba/main/chipinterface.cpp

```diff
@@ -36,6 +36,13 @@
 #include <setup_payload/QRCodeSetupPayloadGenerator.h>
 #include <support/CHIPMem.h>
 
+#if CONFIG_ENABLE_OTAR
```

Comment:
CONFIG_ENABLE_OTA_REQUESTER

Reply:
Noted, I will replace the 'CONFIG_ENABLE_OTAR' to 'CONFIG_ENABLE_OTA_REQUESTER'.